README.md

```diff
@@ -46,7 +46,17 @@ class RecordLoader < GraphQL::Batch::Loader
 end
 ```
 
-Use lazy execution and instrumentation by `GraphQL::Batch` in your schema
+Use `GraphQL::Batch` as a plugin in your schema (for graphql > `1.5.0`).
```

Comment:
I think you could still say `>= 1.5.0`, since it is still true.  Yanking a version doesn't change history.